core/src/main/java/io/kestra/core/utils/TestsUtils.java

```diff
@@ -94,8 +94,8 @@ public static Flow mockFlow() {
 
     private static Flow mockFlow(StackTraceElement caller) {
         return Flow.builder()
-            .namespace(caller.getClassName())
-            .id(caller.getMethodName())
+            .namespace(caller.getClassName().toLowerCase())
+            .id(caller.getMethodName().toLowerCase())
```

Comment:
Why is it needed?

Reply:
Because class names are uppercase so namespace is not compliant with flow namespace validator and in my case (Reconcile Git task which takes everything on git and put it in Kestra), I change retrieved flows' namespace to the currently running one and import them, but it won't work without this change